Sources/SwiftParserDiagnostics/ParserDiagnosticMessages.swift

```diff
@@ -360,6 +360,12 @@ public struct TryCannotBeUsed: ParserError {
   }
 }
 
+public struct PoundElseSpaceIfTypo: ParserError {
+  public var message: String {
+    "error: unexpected 'if' keyword following '#else' conditional compilation directive; did you mean '#elseif'?"
+  }
+}
+
```

Comment:
Since this error doesn’t contain any variables, you can also put it in `StaticParserErrors`, which is a little more succinct.

Reply:
Done